packages/remote/src/electron-browser/remote-frontend-module.ts

```diff
@@ -65,4 +70,19 @@ export default new ContainerModule((bind, _, __, rebind) => {
     bind(RemotePortForwardingProvider).toDynamicValue(ctx =>
         ServiceConnectionProvider.createLocalProxy<RemotePortForwardingProvider>(ctx.container, RemoteRemotePortForwardingProviderPath)).inSingletonScope();
 
+    bind(LocalRemoteFileSytemServer).toDynamicValue(ctx =>
+        isRemoteConnection() ?
+            ServiceConnectionProvider.createLocalProxy(ctx.container, remoteFileSystemPath, new RemoteFileSystemProxyFactory()) :
+            ctx.container.get(RemoteFileSystemServer));
+    bind(LocalEnvVariablesServer).toDynamicValue(ctx =>
+        isRemoteConnection() ?
+            ServiceConnectionProvider.createLocalProxy<EnvVariablesServer>(ctx.container, envVariablesPath) :
+            ctx.container.get(EnvVariablesServer));
+    bind(LocalRemoteFileSystemProvider).toSelf().inSingletonScope();
+    rebind(UserStorageContribution).to(RemoteUserStorageContribution);
```

Comment:
It's not really a breaking change, but if adoptors already rebound the `UserStorageContribution` somewhere they will probably not notice this change and not consume this fix. Do you have an idea on how we could document this? Or do you think this is uneccessary?